### PR TITLE
Fix docstring of FileLink to match parameter name

### PR DESCRIPTION
**Pull request by StevenMaude**
It used to be `directory_prefix`, but is currently `url_prefix`.

### CODE REVIEW

File: IPython/lib/display.py [hunk 1]
```diff
@@ -323,7 +323,7 @@ def __init__(self,
         ----------
         path : str
             path to the file or directory that should be formatted
-        directory_prefix : str
+        url_prefix : str
             prefix to be prepended to all files to form a working link [default:
             'files']
```
Comment (takluyver) [inline IPython/lib/display.py]: The default also appears to have changed to `''`

Comment (StevenMaude) [inline IPython/lib/display.py]: @takluyver Just playing around with FileLink now, and spotted that too.

Will add to this PR.

File: IPython/lib/display.py [hunk 2]
```diff
@@ -327,7 +327,7 @@ def __init__(self,
             prefix to be prepended to all files to form a working link [default:
             '']
         result_html_prefix : str
-            text to append to beginning to link [default: none]
+            text to append to beginning to link [default: '']
```
Comment (StevenMaude) [inline IPython/lib/display.py]: Changed this one for consistency with `url_prefix` docstring update.

First time looking at IPython code, so don't know if the use of `none` here is idiomatic or not. If not, I can drop this commit from the PR.

Comment (takluyver) [inline IPython/lib/display.py]: Thanks, I prefer the precision of what you changed it to.

**Comment by takluyver** [comment]
Thanks for your first contribution to IPython! :cookie:

**Comment by Carreau** [comment]
@meeseeksdev backport to 5.x

**Comment by Carreau** [comment]
@meeseeksdev backport to 5.x

**Comment by Carreau** [comment]
> backport to 5.x

He can't because the fork is gone. I need to fix that at some point.

**Pull request merged**